docs/source/operations/indy_verify.md

```diff
@@ -1,6 +1,16 @@
 # Verify that Indy DLT/Blockchain network is working
-1. You need to verify that network is successfully configured as mentioned in [setting_dlt.md](./setting_dlt.md).
-2. Verify that there are no errors in the logs of stewards.
-2. Connect to cluster using generated pool-genesis. Readme for pool genesis is [here](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/master/platforms/hyperledger-indy/configuration/roles/create/helm_component/pool_genesis/).
-3. Execute multiple commands using CLI for indy-SDK. Readme how to install and use indy CLI is [here](https://github.com/hyperledger/indy-sdk/tree/master/cli).
+1. Recommend you to go through the [Flux Troubleshooting Guide](./flux_troubleshooting.md), before you start your network deployment.
```

Comment:
This cannot be checked-in unless this linked file is present

Reply:
done